src/components/Person/Person.jsx

```diff
@@ -1 +1,31 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export function Person(person) {
+  const { name, age = null, sex, isMarried, partnerName } = person.person;
+  const merried = sex === 'f' ? 'husband' : 'wife';
```

Comment:
```suggestion
  const married = sex === 'f' ? 'husband' : 'wife';
```

Reply:
Not resolved. Fix spell. 

merried -> married